locale/en_US/translations.ts

```diff
@@ -314,6 +314,106 @@
+            <translation>Monday</translation>
+            <extracomment>Day of Week</extracomment>
+        </message>
+        <message>
+            <source>Tuesday</source>
+            <translation>Tuesday</translation>
+            <extracomment>Day of Week</extracomment>
+        </message>
+        <message>
+            <source>Wednesday</source>
+            <translation>Wednesday</translation>
+            <extracomment>Day of Week</extracomment>
+        </message>
+        <message>
+            <source>Thursday</source>
+            <translation>Thursday</translation>
+            <extracomment>Day of Week</extracomment>
+        </message>
+        <message>
+            <source>Friday</source>
+            <translation>Friday</translation>
+            <extracomment>Day of Week</extracomment>
+        </message>
+        <message>
+            <source>Saturday</source>
+            <translation>Saturday</translation>
+            <extracomment>Day of Week</extracomment>
+        </message>
+        <message>
+            <source>Started at</source>
+            <translation>Started at</translation>
+            <extracomment>(Past Tense) For defining time when a program started today (e.g.  Started at 08:00) </extracomment>
+        </message>
+        <message>
+            <source>Started</source>
+            <translation>Started</translation>
+            <extracomment>(Past Tense) For defining a day and time when a program started (e.g.  Started Wednesday, 08:00) </extracomment>
+        </message>
+        <message>
+            <source>Starts at</source>
+            <translation>Starts at</translation>
+            <extracomment>(Future Tense) For defining time when a program will start today (e.g.  Starts at 08:00) </extracomment>
+        </message>
+        <message>
+            <source>Starts</source>
+            <translation>Starts</translation>
+            <extracomment>(Future Tense) For defining a day and time when a program will start (e.g.  Starts Wednesday, 08:00) </extracomment>
+        </message>
+        <message>
+            <source>Ended at</source>
+            <translation>Ended at</translation>
+            <extracomment>(Past Tense) For defining time when a program will ended (e.g.  Ended at 08:00) </extracomment>
+        </message>
+        <message>
+            <source>Ends at</source>
+            <translation>Ends at</translation>
+            <extracomment>(Past Tense) For defining a day and time when a program ended  (e.g.  Ended Wednesday, 08:00) </extracomment>
+        </message>
+        <message>
+            <source>Live</source>
+            <translation>Live</translation>
+            <extracomment>If TV Show is being broadcast live (not pre-recorded)</extracomment>
+        </message>
+        <message>
+            <source>Repeat</source>
+            <translation>Repeat</translation>
+            <extracomment>If TV Shows has previoulsy been broadcast</extracomment>
```

Comment:
```suggestion
            <extracomment>If TV Shows has previously been broadcasted</extracomment>
```
Consistency for Weblate